panther_analysis_tool/main.py

```diff
@@ -1761,16 +1761,22 @@ def run() -> None:
     if getattr(args, "filter_inverted", None) is None:
         args.filter_inverted = {}
 
+    for key in os.environ:
+        if key.startswith("PANTHER_"):
+            logging.info(
+                "Found Environment Variables prefixed with 'PANTHER'.  NOTE: ENVIRONMENT VARIABLES OVERRIDE COMMAND LINE OPTIONS"
```

Comment:
can we invert this? i think it would be more expected that cli opts take precedence

Reply:
aws cli precedence, for example:

<img width="1419" alt="image" src="https://user-images.githubusercontent.com/806620/181858469-ee8b19eb-5b31-4054-9b92-41d3dc1d82d0.png">

can fish up more if necessary

Reply:
updated.

Best I can do is command line > environment > panther_config (these last two are set by dynaconf)